Gelf test timing out

@loicmathieu so we just add this failure for native tests in a Hibernate ORM PR:
```
2020-11-18 14:48:15,298 INFO  [io.qua.log.gel.it.GelfLogHandlerResource] (executor-thread-1) Some useful log message
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 13.362 s <<< FAILURE! - in io.quarkus.logging.gelf.it.GelfLogHandlerIT
Error:  io.quarkus.logging.gelf.it.GelfLogHandlerIT.test  Time elapsed: 12.094 s  <<< ERROR!
org.awaitility.core.ConditionTimeoutException: 
Assertion condition defined as a io.quarkus.logging.gelf.it.GelfLogHandlerTest 1 expectation failed.
Expected status code <200> but was <400>.
 within 10 seconds.
Caused by: java.util.concurrent.TimeoutException
```

https://github.com/quarkusio/quarkus/pull/13352/checks?check_run_id=1418454217

I don't know if we just need to set a higher timeout or if that means we won't be able to test things for Gelf?

--- a/integration-tests/logging-gelf/pom.xml
+++ b/integration-tests/logging-gelf/pom.xml
@@ -151,6 +151,8 @@
                                             <name>graylog</name>
                                             <alias>elasticsearch</alias>
                                         </network>
+                                        <cpus>2</cpus>
+                                        <cpuShares>2048</cpuShares>
                                         <ports>
                                             <port>9200:9200</port>
                                         </ports>
@@ -178,6 +180,7 @@
                                             <name>graylog</name>
                                             <alias>mongo</alias>
                                         </network>
+                                        <cpus>1</cpus>
                                         <ports>
                                             <port>27017:27017</port>
                                         </ports>
@@ -206,6 +209,7 @@
                                         <env>
                                             <GRAYLOG_HTTP_EXTERNAL_URI>http://127.0.0.1:9000/</GRAYLOG_HTTP_EXTERNAL_URI>
                                         </env>
+                                        <cpus>1</cpus>
                                         <ports>
                                             <port>9000:9000</port>
                                             <port>12201:12201/udp</port>

--- a/integration-tests/logging-gelf/src/test/java/io/quarkus/logging/gelf/it/GelfLogHandlerTest.java
+++ b/integration-tests/logging-gelf/src/test/java/io/quarkus/logging/gelf/it/GelfLogHandlerTest.java
@@ -48,14 +48,14 @@ public class GelfLogHandlerTest {
         }
 
         // wait for the input to be running
-        await().during(5, TimeUnit.SECONDS);
+        await().during(10, TimeUnit.SECONDS);
     }
 
     @Test
     public void test() {
         RestAssured.given().when().get("/gelf-log-handler").then().statusCode(204);
 
-        await().atMost(10, TimeUnit.SECONDS)
+        await().atMost(20, TimeUnit.SECONDS)
                 .untilAsserted(
                         () -> {
                             RestAssured.given()

